great-idea/css/index.css

```diff
@@ -62,3 +62,141 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+
+/* ~~~~~~~~~~~~~~~ My Code ~~~~~~~~~~~~~~~ */
```

Comment:
Your code is organized so neatly! Adding section headers to break up your code into sensible, easily identified chunks is a really good habit to get into, and you've done a very nice job of keeping related styles together and organizing your elements/classes in order of increasing specificity. Keep it up!